src/main/java/frc/robot/commands/RotateVectorDriveCommand.java

```diff
@@ -57,10 +65,17 @@ public void execute() {
     double y = translationYSupplier.getAsDouble();
     double rotX = rotationXSupplier.getAsDouble();
     double rotY = rotationYSupplier.getAsDouble();
+    boolean isRobotRelative = isRobotRelativeSupplier.getAsBoolean();
 
     // if stick magnitude is greater then rotate angle mag
```

Comment:
this comment stayed in the same place, but the code it was referring to moved -- please dont do that